AoC/Day01/Day1Solver.cs

```diff
@@ -1,16 +1,18 @@
+using static System.Environment;
+
 namespace AoC.Day01;
 
 public class Day1Solver : SolverBase
 {
-    public override string DayName => "";
+    public override string DayName => "Calorie Counting";
+
+    private static IEnumerable<long> ParseElves(PuzzleInput input) =>
+        input.ToString()
+            .Split($"{NewLine}{NewLine}")
```

Comment:
Nit pick: This could be just:

```suggestion
            .Split(NewLine + NewLine)
```

String concat of 2 strings of the known small size will be less expensive than using a string builder (which string interpolation uses behind the scenes); and its also slightly less chars!